src/modules/rover_ackermann/module.yaml

```diff
@@ -141,6 +141,21 @@ parameters:
         decimal: 2
         default: 5
 
+      RA_VEL_RED_START:
+        description:
+          short: Scaling factor on the distance to the WP at which the cornering slow down effect starts
+          long: |
+            This is a multiplicative factor on the acceptance radius of the waypoints.
+            A value higher than one means that the rover will start to slow down before it reaches the
```

Comment:
Would be great to get the exact mathematical relationship between this param, the distance to the WP and the commanded speed here in the param description.

Reply:
Changed in [f8f7084](https://github.com/PX4/PX4-Autopilot/pull/23310/commits/f8f70842f2da6676f5ac360dd19987efad4c950b)